src/dev/file.ts

```diff
@@ -81,4 +88,8 @@ export class File {
   public toJSON() {
     return this.relativePath;
   }
+
+  public getContent() {
+    return this.getContentCB ? this.getContentCB() : undefined;
```

Comment:
I think this should throw if the file isn't virtual, maybe `getVirtualContent()`?